contracts/BSCValidatorSet.sol

```diff
@@ -83,6 +83,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   uint256 public finalityRewardRatio;
   uint256 public previousHeight;
   uint256 public previousBalanceOfSystemReward;
+  bytes[] public previousVoteAddrsSet;
```

Comment:
`previousVoteAddrsSet` typo? Get warning in my IDE

Reply:
> `previousVoteAddrsSet` typo? Get warning in my IDE

previousVoteAddrsSet -->previousVoteAddrSet ?
but all with warning , because it's not a existed word?